packages/strapi-plugin-content-type-builder/admin/src/components/List/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+*
+* List
+*
+*/
+
+import React from 'react';
+import { forEach, has, map, size } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import ButtonPrimaryHotline from 'components/Button';
+import AttributeRow from 'components/AttributeRow';
+import styles from './styles.scss';
+
+class List extends React.Component { // eslint-disable-line react/prefer-stateless-function
+
+  renderListTitle = () => {
+    const availableNumber = size(this.props.model.attributes);
+    const title = availableNumber > 1 ? 'modelPage.contentType.list.title.plural'
+      : 'modelPage.contentType.list.title.singular';
+
+    let relationShipNumber = 0;
+
+    forEach(this.props.model.attributes, (attribute) => {
```

Comment:
Pro tips: 
```js
const relationShipNumber = this.props.models.attributes.filter(attr => has(attr.params, 'model')).length;
```

It avoids initializing a variable out of the scope, it allows you to use a constant variable and it's shorter.